src/main/java/competition/subsystems/collector/CollectorSubsystem.java

```diff
@@ -120,6 +134,16 @@ public void periodic() {
             if (loopCount % 250 == 0) {
                 log.info("PressureSensorValue:" + pressureSensor.getVoltage());
             }
+
+            if (currentIntakeTime - intakeTime > 0.5) {
```

Comment:
`currentIntakeTime` should be calculated here in periodic, rather than in `intake()`